zerver/lib/error_notify.py

```diff
@@ -23,8 +23,8 @@ def logger_repr(report: Dict[str, Any]) -> str:
     return "Logger {logger_name}, from module {log_module} line {log_lineno}:".format(**report)
 
 def user_info_str(report: Dict[str, Any]) -> str:
-    if report['user_full_name'] and report['user_email']:
-        user_info = "{user_full_name} ({user_email})".format(**report)
+    if report['user']['user_info_exist']:
```

Comment:
Why do we need `user_info_exist` here?  Can't it just be `if role.get('user_info') is not None`?

Reply:
Oh, indeed! I've fixed it

Reply:
I don't think you understood my comment -- what I meant was to check whether a key is present in the `report` dictionary, rather than adding a boolean field whose value can be inferred from whether that key exists or not.

Reply:
Aaah ok. I just initially thought that it could be a bit odd to check `report['user'].get('user_full_name')` and below access informations other than `user_full_name`, and doing `report['user'].get('user_info_exist')` makes it more clear that it's guaranteed that all the other info is present.